### ISSUE TITLE
Is it still maintained?

### ISSUE THREAD

**Issue by eserte** [simra-project/dataset#7]
Currently there is no newer data than 2024-03 available.

**Comment by selphiron** [comment]
Yes, it is. I need to update the dataset though, that is correct. Thanks for pointing that out.

**Comment by selphiron** [comment]
> Currently there is no newer data than 2024-03 available.

I updated the dataset:
[DepositOnce (08/2023-03/2024)](https://doi.org/10.14279/depositonce-20446)  
[DepositOnce (04/2024-01/2025)](https://doi.org/10.14279/depositonce-22968)
